conductr_cli/sandbox_init.py

```diff
@@ -0,0 +1,51 @@
+from conductr_cli import docker_machine, terminal
+import logging
+from subprocess import CalledProcessError
+
+
+def init(args):
+    """`sandbox init` command"""
+
+    log = logging.getLogger(__name__)
+    vm_name = docker_machine.vm_name()
```

Comment:
Don't we always want to check for the conductr vmname here ie not just the current one?